cloud-agent/service/server/src/main/scala/org/hyperledger/identus/presentproof/controller/http/RequestPresentationInput.scala

```diff
@@ -128,6 +132,20 @@ object RequestPresentationInput {
             )
           )
         )
+
+    object presentationFormat
+        extends Annotation[Option[String]](
+          description =
+            "The presentation format to display in Didcomm messages (default to 'prism/jwt', vc+sd-jwt or anoncreds/proof-request@v1.0)",
+          example = Some("prism/jwt"),
+          validator = Validator.enumeration(
+            List(
+              Some("prism/jwt"),
```

Comment:
@CryptoKnightIOG, could you clarify where you got the names of the formats `prism/jwt` and `vc+sd-jwt`?